<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>400</width>
      <height>400</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>70</y>
      <width>312</width>
      <height>251</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="5" column="0" colspan="3">
      <widget class="QTextBrowser" name="textBrowser"/>
     </item>
     <item row="2" column="2" alignment="Qt::AlignRight">
      <widget class="QPushButton" name="reset_button">
       <property name="text">
        <string>Reset</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0" alignment="Qt::AlignRight">
      <widget class="QPushButton" name="ready_button">
       <property name="text">
        <string>Ready</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" alignment="Qt::AlignRight">
      <widget class="QLabel" name="seed_label">
       <property name="text">
        <string>Seed value:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" alignment="Qt::AlignRight">
      <widget class="QLabel" name="goal_label">
       <property name="text">
        <string>Goal (power of 2):</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1" alignment="Qt::AlignRight">
      <widget class="QPushButton" name="pause_button">
       <property name="text">
        <string>Pause</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" colspan="2">
      <widget class="QSpinBox" name="goal_spinbox"/>
     </item>
     <item row="0" column="1" colspan="2">
      <widget class="QSpinBox" name="seed_spinbox"/>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="close_button">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>390</y>
      <width>80</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Close</string>
    </property>
   </widget>
   <widget class="QPushButton" name="right_button">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>370</y>
      <width>45</width>
      <height>45</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="left_button">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>370</y>
      <width>45</width>
      <height>45</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="down_button">
    <property name="geometry">
     <rect>
      <x>525</x>
      <y>415</y>
      <width>45</width>
      <height>45</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="up_button">
    <property name="geometry">
     <rect>
      <x>525</x>
      <y>325</y>
      <width>45</width>
      <height>45</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>10</y>
      <width>311</width>
      <height>60</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="1" rowspan="2">
      <widget class="QLCDNumber" name="minute_lcdnumber"/>
     </item>
     <item row="0" column="0" rowspan="2" alignment="Qt::AlignRight|Qt::AlignVCenter">
      <widget class="QLabel" name="time_label">
       <property name="text">
        <string>Time used:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2" rowspan="2">
      <widget class="QLCDNumber" name="second_lcdnumber"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
